README.md

```diff
@@ -17,29 +17,58 @@ Requirements
 Permissions to Google Cloud API. If running on an old Compute Engine instance or
 Compute Engine instances created without the default credentials, then you must
 complete the following steps
-https://cloud.google.com/monitoring/agent/authorization#before_you_begin
+https://cloud.google.com/monitoring/agent/authorization#before_you_begin.
```

Comment:
I think this URL is broken with the trailing . on it

Reply:
Seems to be working when I go to https://github.com/GoogleCloudPlatform/stackdriver-ansible-role/tree/rmoriar-add-uninstall-via-script, will remove if the link doesnt work after merging.